### PR TITLE
Added the step2 JavaScript (dynamical content)

### PR DESCRIPTION
**Pull request by tzavidas**

### CODE REVIEW

File: src/main/webapp/renderers/Step2Renderer.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    generateInfoPane(prevEvent, nextEvent, index) {
+        if(prevEvent.location == nextEvent.location) {
+            return `
+                <div class="info-pane">
+                    <p class="same-location-msg">Same location</p>
+                </div>
+            `;
+        } else {
```
Comment (maxwbuckley) [inline src/main/webapp/renderers/Step2Renderer.js]: Can we remove this else as we are saying if "a" return something. So by default the rest is not "a"

File: src/main/webapp/renderers/Step2Renderer.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import RendererInterface from './RendererInterface.js';
+
+export default class Step2Renderer extends RendererInterface {
+    constructor(parentElement) {
+        super(parentElement);
+    }
+
+    generateInfoPane(prevEvent, nextEvent, index) {
+        if(prevEvent.location == nextEvent.location) {
+            return `
```
Comment (maxwbuckley) [inline src/main/webapp/renderers/Step2Renderer.js]: Nit: Could we make this string a constant with a meaningful name?

Same idea could be applied below

File: src/main/webapp/renderers/Step2Renderer.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+            this.parentElement.innerHTML += `
+                <div class="event-container">
+                    <h3>${event.name}</h3>
+                    <p>${event.startingTime} - ${event.endingTime}</p>
+                    <p>${event.location}</p>
+                </div>
+            `;
+
+            if(i < events.length - 1) { // is an intermediate event
```
Comment (semenovm) [inline src/main/webapp/renderers/Step2Renderer.js:43]: Is it separator?

**Review by semenovm — COMMENTED** [review]
As I mentioned earlier (during codelabs) using innerHTML is far from the best solution, but let's keep it as is.

**Review by maxwbuckley — APPROVED** [review]

File: src/main/webapp/renderers/Step2Renderer.js (hunk 2 above)
Comment (tzavidas) [reply src/main/webapp/renderers/Step2Renderer.js]: Alright, I will combine them into a nice ?: operator!

File: src/main/webapp/renderers/Step2Renderer.js (hunk 3 above)
Comment (tzavidas) [reply src/main/webapp/renderers/Step2Renderer.js:43]: What do you mean by a seperator?

File: src/main/webapp/renderers/Step2Renderer.js (hunk 2 above)
Comment (tzavidas) [reply src/main/webapp/renderers/Step2Renderer.js]: Done! Can you have a look?

File: src/main/webapp/renderers/Step2Renderer.js (hunk 1 above)
Comment (tzavidas) [reply src/main/webapp/renderers/Step2Renderer.js]: Done! Can you have a look?

File: src/main/webapp/renderers/Step2Renderer.js (hunk 3 above)
Comment (semenovm) [reply src/main/webapp/renderers/Step2Renderer.js:43]: Does this code block renders an element that separates two events?

Comment (tzavidas) [reply src/main/webapp/renderers/Step2Renderer.js:43]: Exactly! Does this look good? Can I merge it?

**Review by semenovm — APPROVED** [review]

**Pull request merged**